src/manager.rs

```diff
@@ -811,6 +817,26 @@ impl<C: Store> Manager<C, Registered> {
         Ok(group_changes)
     }
 
+    pub fn save_group(&self, group:Group, key:Vec<u8>) -> Result<(), Error> {
```

Comment:
```suggestion
    pub fn save_group(&self, key: impl AsRef<[u8]>, group: Group) -> Result<(), Error> {
```

Reply:
here we have to rethink, it a little. I set the `public_key: key` which needs an vector. I misuse that field to pass the group id to `self.config_store.save_group(proto_group)?;` We have to either make a struct à la {key, group} or we need to pass the key as second argument.